pypeit/coadd2d.py

```diff
@@ -1370,21 +1401,22 @@ def compute_offsets(self, offsets):
                 It could be a list of offsets, or a string, or None. If equal to 'maskdef_offsets' the
                 offsets computed during the slitmask design matching will be used.
         """
-        super().compute_offsets(offsets)
+        super().compute_offsets()
 
         # adjustment for multislit to case 4) parset `offsets` = 'auto' but we have a bright object
-        if offsets == 'auto' and self.objid_bri is not None:
+        if self.par['coadd2d']['offsets'] == 'auto' and self.obj_id_bri is not None:
```

Comment:
The rest of this function is inside this if statement.  Can we instead return if these conditions are not met?